src/main/java/org/benetech/servicenet/service/dto/provider/ProviderOpeningHoursDTO.java

```diff
@@ -0,0 +1,86 @@
+package org.benetech.servicenet.service.dto.provider;
+
+import java.io.Serializable;
+import java.util.Objects;
+import java.util.UUID;
+import javax.validation.constraints.NotNull;
+
+/**
+ * A DTO for the OpeningHours entity.
+ */
+public class ProviderOpeningHoursDTO implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private UUID id;
+
+    @NotNull
+    private Integer weekday;
+
+    private String opensAt;
+
+    private String closesAt;
+
+    public UUID getId() {
```

Comment:
Here you use getters/setters and in another case Lombok's `@Data` and in yet another case Lombok's `@Getters/@Setters`. Perhaps we should be more consistent with this.